libraries/commerce/cart/streams/index.js

```diff
@@ -5,7 +5,10 @@ import {
   routeWillLeave$,
   routeDidLeave$,
 } from '@shopgate/pwa-common/streams/router';
-import { HISTORY_PUSH_ACTION } from '@shopgate/pwa-common/constants/ActionTypes';
+import {
+  HISTORY_PUSH_ACTION,
```

Comment:
Instead of the HISTORY... actions, please use `import { ACTION_PUSH, ACTION_REPLACE } from '@virtuous/conductor';` please, the history stuff is deprecated and should soon be removed :/ Sorry.